src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1418,9 +1436,6 @@ public void onDisable() {
                 // kill channel topic updater
                 if (channelTopicUpdater != null) channelTopicUpdater.interrupt();
 
-                // kill console message queue worker
-                if (consoleMessageQueueWorker != null) consoleMessageQueueWorker.interrupt();
-
```

Comment:
The appender should be shutdown at disable

Reply:
[It is](https://github.com/DiscordSRV/DiscordSRV/pull/1309/files#diff-cafc7a5fe69034c557bfa518b2df149d2d01fbaa7064a31f5aeb8019ae95471eR1458-R1459)

Reply:
Link goes to nowhere, but I see